php/7.0/Dockerfile

```diff
@@ -21,36 +22,34 @@ RUN set -ex; \
 		libxml2-dev \
 		libpng-dev \
 		libzip-dev \
-		mariadb-client \
-		msmtp \
+		libssl-dev \
+		libicu-dev \
 		unzip \
 		vim \
 		zip
 
-RUN pecl install imagick memcached; \
-	printf "\n" | pecl install mcrypt-1.0.1; \
-	printf "\n" | pecl install redis; \
-	docker-php-ext-configure gd --enable-gd-native-ttf --with-freetype-dir=/usr/include/freetype2 --with-png-dir=/usr/include --with-jpeg-dir=/usr/include; \
-	docker-php-ext-configure zip --with-libzip; \
-	docker-php-ext-configure --with-php-config=/usr/local/bin/php-config; \
-	docker-php-ext-configure imap --with-kerberos --with-imap-ssl; \
-	docker-php-ext-install gd imap mcrypt mysqli opcache soap zip; \
+RUN pecl install imagick; \
+	pecl install memcached; \
+	pecl install mcrypt-1.0.3; \
+	pecl install redis; \
+	docker-php-ext-configure gd --with-freetype --with-jpeg; \
```

Comment:
@mrrobot47 Just check the output of this command while building it to ensure that gd is configured correctly with freetype, png and jpeg. I passed additional arguments as one of rtcamp's project was having trouble where captcha was being generated which needed freetype to work properly.

So AFAIK earlier the arguments were `--with-freetype --with-jpeg` and I changed it to `--enable-gd-native-ttf --with-freetype-dir=/usr/include/freetype2 --with-png-dir=/usr/include --with-jpeg-dir=/usr/include`.

You can check if these flags are still needed or not by diff-ing the output of commands with old and new flags.

Reply:
Same comment for the rest of the dockerfiles

Reply:
Thanks a lot for the input. Will get this checked :+1:

Reply:
The changes were made only for the PHP-7.4 image initially. The rest of the images might need changes according to their PHP version.

Reference:
https://github.com/php/php-src/blob/PHP-7.4/UPGRADING#L761
https://github.com/EasyEngine/dockerfiles/pull/97#issue-396338844

Reply:
@kirtangajjar Thanks for letting us know about this. https://github.com/EasyEngine/dockerfiles/pull/99/commits/631783c5c00d7b409d3eaba4739dd6ee81bcba39 resolves it.